### PR TITLE
add support for new slack post format and multiple attachments in slack importer

### PR DESCRIPTION
**Pull request by svelle**
The previous PR added support for importing private messages etc. but could not correctly parse the new message attachment format.

This PR adds support for said format by parsing the files array as well as keeping backwards compatibility to the old format and thus enables compatibility to the advanced-slack-exporter by @grundleborg when this PR get's merged: https://github.com/grundleborg/slack-advanced-exporter/pull/10

Hopefully this will be the last commit needed for full corporate import ability *including* attachments.

Edit: I noticed one bug while testing though and would appreciate any hints in the right direction. 
Uploaded pictures don't have a thumbnail, is there any way to regenerate those for the imported pictures?

### CODE REVIEW

**Comment by svelle** [comment]
@grundleborg I tried with an fresh export I did a few days ago and it seemed to work so far.
If you say it looks good to you I'd send this to a customer that's been waiting for a custom build and wait for feedback from them before merging. Sounds good?

**Comment by grundleborg** [comment]
@svelle sounds good to me.

File: tests/slack-import-test-posts.json [hunk 1]
```diff
@@ -176,5 +176,41 @@
         "username": "<@U07Q4MHCP|lindy>",
         "bot_id": null,
         "ts": "1469782077.000010"
+    },
+    {
+        "type": "message",
+        "text": "and an archive",
+        "files": [
+            {
```
Comment (lieut-data) [inline tests/slack-import-test-posts.json]: nit: Could we test `files` with more than one entry in the array?

Comment (svelle) [reply tests/slack-import-test-posts.json]: hey @lieut-data I replaced this with a new post from a fresh export that has more than one attachment.

I also added an additional test for the attachments, to test if the correct number of attachments was parsed. lmk if you have any suggestions for improvements on this.

**Comment by amyblais** [comment]
@svelle Please help create a Jira Story ticket for this too.

**Comment by svelle** [comment]
@amyblais testing steps are the same as with the other PR :)

**Pull request merged**